spec/support/custom_matchers.rb

```diff
@@ -0,0 +1,11 @@
+RSpec::Matchers.define :have_submit_button do |value|
```

Comment:
Would it be possible to add these custom matchers as capybara selectors instead? like we've done with the test id?

Reply:
I may be missing something, as I do not have much experience customizing Capybara. However, based on the research that I did, it seemed to make the most sense to add helper methods that utilize the custom capybara selector that we already had. Take a look at the new commit and let me know if I should have done this differently.